package/nix/generate_derivation.sh

```diff
@@ -1,20 +1,17 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p git haskellPackages.cabal2nix nix-prefetch-git jq
+#! nix-shell -i bash -p git cabal2nix nix-prefetch-git jq
```

Comment:
`haskellPackages.cabal2nix` seemed to have a ton (~15GB) of dependencies that it pulled in, but switching to just `cabal2nix` apparently doesn't need them 🤷‍♀️